internal/observability/observability.go

```diff
@@ -16,75 +16,56 @@
 package observability
 
 import (
-	"os"
-	"strings"
-	"sync"
+	"context"
+	"fmt"
 
 	"contrib.go.opencensus.io/exporter/ocagent"
 	"contrib.go.opencensus.io/exporter/stackdriver"
-	"go.opencensus.io/plugin/ocgrpc"
-	"go.opencensus.io/plugin/ochttp"
-	"go.opencensus.io/stats/view"
-	"go.opencensus.io/trace"
 )
 
-func init() {
-	initExporterOnce.Do(initExporter)
+// Exporter defines the minimum shared functionality for an observability exporter
+// used by this application.
+type Exporter interface {
+	InitOnce()
```

Comment:
Can we use a less common thing here like `InitExport()` just to avoid a collision with another iface?

Reply:
Sure. Does `InitExportOnce()` make sense to you? I think the "once" part is important.

Reply:
What would happen if it were invoked twice?